beetsplug/gmusic.py

```diff
@@ -38,6 +38,12 @@ def __init__(self):
         else:
             self.m.perform_oauth()
 
+        self.config.add({
+            u'auto': False,
+        })
+        if self.config['auto'].get(bool):
```

Comment:
The `.get(bool)` bit is not actually necessary here—configuration views get coerced to Booleans automatically.